src/routes/web.php

```diff
@@ -43,6 +43,8 @@
     Route::post('/add-comment', [PostCommentController::class, 'create'])->name('post.comment.create');
     Route::delete('/delete-comment', [PostCommentController::class, 'delete']);
     Route::put('/edit-comment', [PostCommentController::class, 'update']);
+    Route::put('/edit-comment', [PostCommentController::class, 'update']);
```

Comment:
why other routes no route name?

Reply:
can we do the naming for the routes in another PR?

Reply:
noted